mobile-app/lib/ui/views/learn/widgets/quiz_widget.dart

```diff
@@ -175,7 +178,11 @@ class QuizWidget extends StatelessWidget {
       feedbackWidgets.addAll(
         parser.parse(
           feedback,
-          fontColor: isCorrect == true ? FccColors.green40 : FccColors.red15,
+          customStyles: {
+            '*:not(h1):not(h2):not(h3):not(h4):not(h5):not(h6)': Style(
```

Comment:
Same here, I do not think we have any headers. 

Maybe exclude `pre` and `code`?